ivy_tests/test_ivy/test_frontends/test_numpy/test_linalg/test_matrix_eigenvalues.py

```diff
@@ -93,6 +93,55 @@ def test_numpy_eig(
     )
 
 
+# eigvals
+@handle_frontend_test(
+    fn_tree="numpy.linalg.eigvals",
+    dtypes_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=10,
+        shape=helpers.ints(min_value=2, max_value=4).map(lambda x: tuple([x, x])),
+    ).filter(
+        lambda x: "float16" not in x[0]
+        and "bfloat16" not in x[0]
+        and np.linalg.cond(x[1][0]) < 1 / sys.float_info.epsilon
+        and np.linalg.det(np.asarray(x[1][0])) != 0
+    ),
+    test_with_out=st.just(False),
+)
+def test_numpy_eigvals(
+    dtypes_values,
+    frontend,
+    test_flags,
+    fn_tree,
+    backend_fw,
+    on_device,
+):
+    dtypes, a = dtypes_values
+    ret, frontend_ret = helpers.test_frontend_function(
+        input_dtypes=dtypes,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        test_values=False,
+        a=a,
+    )
+
+    ret = np.sort(to_numpy(ret)).astype(np.complex128)
```

Comment:
Hey @moamen20, can you please rectify the failing tests. There is no such thing as  `to_numpy`